File: JS PS/q3.js
// 3. Write a function that takes three sides of a triangle and checks whether the triangle is equilateral, isosceles, or scalene.

function check_triangle(side1, side2, side3) {
    
    if (side1 === side2 && side2 === side3) {
        console.log('Equilateral Triangle');
    }
    else if (side1 === side2 || side2 === side3 || side1 === side3) {
        console.log('Isosceles Triangle');
    }
    else {
        console.log('Scalene Triangle');
    }
}

File: Clang PS/q3.c
// 3. Write a function that takes three sides of a triangle and checks whether the triangle is 
// equilateral, isosceles, or scalene.

#include <stdio.h>

int main() {
    int side1, side2, side3;   
    printf("Enter the lengths of the three sides (comma-separated): ");
    scanf("%d, %d, %d", &side1, &side2, &side3);  
    if(side1==side2 && side2 == side3){
        printf("Equilateral \n");

    }
    else if(side1 == side2 || side2 == side3){
        printf("Isoceles \n");
    }

    else if( side1!= side2 && side2!=side3){
        printf("Scalene \n");
    }


    return 0;
}